msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,135 +62,195 @@ You must configure the `fetchTask` property under `bots.configuration` object in
+
+If you set the `fetchTask` property in the app manifest to:
+
+* **false**: The bot doesn't fetch a dialog or an Adaptive Card. Instead, the bot must provide a static dialog or card that is used when the bot is invoked. For more information, see [dialogs](../../task-modules-and-cards/what-are-task-modules.md).
+
+* **true**: The bot initiates `ConfigFetch` to fetch content. When the bot is invoked, you can return an Adaptive Card or a dialog depending on the context provided in [channelData and userdata](../../messaging-extensions/how-to/action-commands/create-task-module.md#payload-activity-properties-when-a-dialog-is-invoked-from-a-group-chat).
+
+Bot can respond to `ConfigFetch` request in three ways:
+
+1. `type: "continue"`: `type: "continue"` is used to define a continuation of a dialog or Adaptive Card within a bot configuration. When the type is set to `continue`, it indicates that the bot is expecting further interaction from the user to continue with the configuration process.
+
+   The `adaptiveCardForContinue` and `adaptiveCardForSubmit` are custom functions that return the JSON for an Adaptive Card to be used in different stages of a bot’s workflow. These functions are used to return Adaptive Cards for different scenarios based on the user’s interaction with the bot.
+
+   When the user submits the configuration, the `OnTeamsConfigSubmitAsync` method is triggered.  It reads the user's input and returns a different Adaptive Card. You can also update the bot configuration to return a [dialog](../../task-modules-and-cards/what-are-task-modules.md).
+
+   # [C#](#tab/teams-bot-sdk1)
+
+   * [SDK reference](/dotnet/api/microsoft.bot.schema.teams.taskmodulecontinueresponse)
+
+   * [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-configuration-app/csharp/Bot%20configuration/Bots/TeamsBot.cs#L550)
+
+      ```csharp
+         ConfigResponseBase response = new ConfigResponse<TaskModuleResponseBase>
+         {
+            Config = new TaskModuleContinueResponse
+            {
+                  Value = new TaskModuleTaskInfo
+                  {
+                     Height = 123,
+                     Width = 456,
+                     Title = "test title",
+                     Card = new Attachment
+                     {
+                        ContentType = AdaptiveCard.ContentType,
+                        Content = card
+                     }
+                  },
+                  Type = "continue"
+            }
+         };
+      ```   
+
+   # [JavaScript](#tab/JS1)
+
+   * [SDK Reference]
+
+   * [Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-configuration-app/nodejs/teamsBot.js#L83)
+
+      ```javascript
+         const adaptiveCard = CardFactory.adaptiveCard(this.adaptiveCardForContinue());
```

Comment:
I saw many code snippets here are not a compete method, it's copied from partial method. I suggest: either we put full compete method (with method name, input parameter...), or we ensure the "sample code reference" is pointing exactly the same place that showing the code snippets in the sample bot, so the developers can easily understand how to use them w/o confusion.

Reply:
Updated.

Reply:
I understand some methods are very long and you want to copy paste partial of the method, but I saw for the JS code, the method is pretty short, what's the reason to not paste the full method signature in the doc but just the the code after the "response"